Makefile

```diff
@@ -228,7 +228,7 @@ ifneq ($(HAS_WEBAPP),)
 	mkdir -p dist/$(PLUGIN_ID)/webapp
 	cp -r webapp/dist dist/$(PLUGIN_ID)/webapp/
 endif
-	cd dist && tar -cvzf $(BUNDLE_NAME) $(PLUGIN_ID)
+	cd dist/$(PLUGIN_ID) && tar -cvzf ../$(BUNDLE_NAME) .
```

Comment:
I do wonder what prompted this change. The MM server accepts both the previous and the new formats. 

Why the change?